use-async-queue.test.js

```diff
@@ -1,5 +1,5 @@
 import { renderHook, act } from '@testing-library/react-hooks';
-import useAsyncQueue from './use-async-queue';
+import useAsyncQueue from './dist/use-async-queue';
```

Comment:
I converted this file to TS and it compiled, but Jest was complaining about running a TS file and I didn't want to spend the time getting it set up at the moment.